.gitignore

```diff
@@ -11,4 +13,6 @@ pkg/
 .kubernetes-manifests-*/
 .project
 .eclipse.buildship.core.prefs
-release/wi-kubernetes-manifests.yaml
\ No newline at end of file
+.terraform
```

Comment:
this should be `.terraform/*` to ignore the folder

Also I would recommend adding in *`.tfvars` to exclude a user's project data 👍